lpc845-hal/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name    = "lpc845-hal"
+version = "0.4.0"
+authors = [
+    "David Sawatzke <david@sawatzke.de>",
+    "Hanno Braun <hanno@braun-robotics.com>"
+]
+edition = "2018"
+
+description   = "Hardware Abstraction Layer (HAL) for the NXP LPC82x series of ARM Cortex-M0+ microcontroller"
+documentation = "https://docs.rs/lpc845-hal"
+repository    = "https://github.com/lpc-rs/lpc82x-hal"
+license       = "0BSD"
+readme        = "README.md"
+categories    = ["embedded", "hardware-support", "no-std"]
+keywords      = ["arm", "cortex-m", "nxp", "lpc", "embedded-hal-impl"]
+
+
+[badges]
+maintenance = { status = "actively-developed" }
+travis-ci   = { repository = "lpc-rs/lpc82x-hal" }
+
+
+[dependencies]
+cortex-m     = "0.5.2"
+embedded-hal = { version = "0.2", features = ["unproven"] }
+lpc845-pac   = "0.1"
+nb           = "0.1.1"
+void         = { version = "1", default-features = false }
+# This should be in [dev-dependencies], but those can't be optional. Issue:
+# https://github.com/rust-lang/cargo/issues/1596
+compiletest_rs = { version = "0.3", optional = true }
+
+[dependencies.lpc8xx-hal-common]
+path = "../lpc8xx-hal-common"
+features = ["845"]
```

Comment:
I'm pretty sure we won't be able to publish it on crates.io like this, due to the `path` dependency. I believe it's possible to combine `path` and `version` though. Then Cargo will know which crates.io dependency to use when downloading `lpc845-hal` as a dependency, but will use the `path` dependency when building locally.